test/physics/em/ImportedProcesses.test.cc

```diff
@@ -196,6 +197,39 @@ TEST_F(ImportedProcessesTest, photoelectric)
     }
 }
 
+TEST_F(ImportedProcessesTest, bremsstrahlung)
+{
+    // Create photoelectric process (requires Geant4 environment variables)
+    std::shared_ptr<BremsstrahlungProcess> process;
+    try
+    {
+        process = std::make_shared<BremsstrahlungProcess>(
+            particles_, materials_, processes_);
+    }
+    catch (const RuntimeError& e)
+    {
+        SKIP("Failed to create process: " << e.what());
+    }
+
+    // Test model
+    auto models = process->build_models(ModelIdGenerator{});
+    ASSERT_EQ(1, models.size());
+    ASSERT_TRUE(models.front());
+    EXPECT_EQ("Seltzer-Berger bremsstrahlung", models.front()->label());
+    auto all_applic = models.front()->applicability();
+    ASSERT_EQ(2, all_applic.size());
+    Applicability applic = *all_applic.begin();
+
+    // Test step limits
+    for (auto mat_id : range(MaterialId{materials_->num_materials()}))
+    {
+        applic.material = mat_id;
+        auto builders   = process->step_limits(applic);
+        EXPECT_TRUE(builders[VGT::macro_xs]);
+        EXPECT_TRUE(builders[VGT::energy_loss]);
```

Comment:
Can you `EXPECT_FALSE(builders[VGT::range]); ` if that's what the data has? Incidentally, do we understand why there's no range even though there's a dedx? Is it just that the range is so long (de/dx so low) that it can be neglected?

Reply:
I believe there should be a range, yes. So a bit confused by this.

Reply:
@whokion @stognini any idea why brems doesn't have range? I remember this oddity when I was working with the data over Christmas but don't recall finding out why it is the case.

Reply:
See Sec. 7.2.3, 7.5 in G4 Phys. Ref. Manual. The latter seems to convert "cut-in-range" to production thresholds in kinetic energy.

Reply:
I am still looking for a clear answer, but range tables are calculated from dE/dx tables, as far as I remember. Is there any chance that G4 converts E to Range on the fly because it is less expensive than lookup in this case?

Reply:
Looking at `G4LossTableManager` and [section 5.6.4 of this document](http://cds.cern.ch/record/602040/files/it-2002-003.pdf?version=1/#page=48) (though it describes a now obsolete class), it sounds like de/dx tables are constructed by summing the contributions from all energy loss processes (i.e. ionization and bremsstrahlung), and then the range tables are calculated from those de/dx tables. So the ionization range tables might actually account for both ionization and bremsstrahlung?